### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by strugee**

### CODE REVIEW

File: doc/pages/apps/initialization-and-preferences-forms.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ This sample form also makes use of [translation functions](:apps/translations.ht
       entries.push(['string', PORT, 'Port'])
 
       values[COUNTRY_VARIANT] = Nuvola.config.get(COUNTRY_VARIANT)
-      entries.push(['header', _('Amazon Cloud Player')])
+      entries.push(['header', _('Amazon Butt Player')])
```
Comment (fenryxo) [inline doc/pages/apps/initialization-and-preferences-forms.md:270]: This isn't correct.

**Review by jiri-janousek — CHANGES_REQUESTED** [review]
The last correction isn't correct.

File: doc/pages/apps/initialization-and-preferences-forms.md (hunk 1 above)
Comment (strugee) [reply doc/pages/apps/initialization-and-preferences-forms.md:270]: Oh man, so sorry! I'm embarrassed; this wasn't me, it was a silly extension I have installed 😅

I'll push an amended patch shortly.

**Comment by strugee** [comment]
Fixed.

**Review by jiri-janousek — APPROVED** [review]

**Comment by fenryxo** [comment]
Merged.

**Pull request closed without merging**